test/test_jit_cuda_fuser.py

```diff
@@ -418,10 +454,11 @@ def t(x: torch.Tensor):
             return o
 
         try:
-            if data is None:
-                x = torch.randn(shape, dtype=dtype, device="cuda")
+            if random_data:
+                x = torch.randn(shape, dtype=torch.float32, device="cuda")
+                x = x.to(dtype=dtype)
```

Comment:
If it's an int type can we multiply by something like 5 to get a larger variance?

Reply:
Updated. Thanks!